src/transformers/trainer.py

```diff
@@ -3407,8 +3407,6 @@ def _save_tpu(self, output_dir: Optional[str] = None):
         logger.info(f"Saving model checkpoint to {output_dir}")
         model = self.model
         xm.mark_step()
-        if self.args.save_safetensors:
```

Comment:
Will you check if your change works with args.save_safetensors=False?

Reply:
Tests pass with `save_safetensors=False`.